gitlab/commands.yaml

```diff
@@ -0,0 +1,28 @@
+commands:
+  gitlab_stats:
+    sourcePath: packages/gitlab/gitlab_stats.js
+    description: 'View community contribution stats for provided GitLab project repository. Example repository: username/project'
+    parameters:
+      - name: repo
+        optional: false
+  gitlab_create:
+    sourcePath: packages/gitlab/gitlab_create.js
+    description: 'Creates a GitLab ticket with provided project repository, title, and descriptions. Example repository: username/project'
+    parameters:
+      - name: repo
+        optional: false
+      - name: title
+        optional: false
+      - name: description
+        optional: false
+  gitlab_stats:
+    sourcePath: packages/gitlab/gitlab_users.js
+    description: 'View list of users in GitLab project repository. Example repository: username/project'
+    parameters:
+      - name: repo
+        optional: false
+      - name: name
+        optional: true
+  gitlab_gitlab:
```

Comment:
Should just to `gitlab`.  I.E, running `/dapp gitlab` should show the help output

Reply:
Thank you. I like that command name much more. I wanted to make it 'GitLab' or 'help' but didn't want to stray from the app_command guideline.